less/navigation.less

```diff
@@ -1 +1,7 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+.layout (@direction,@align){
+    display:flex;
+    flex-direction: @direction;
+    align-items:@align;
+}
```

Comment:
Now that looks like a parametric mixin to me. This should be in the mixins file. 👍